src/main/java/uk/gov/hmcts/befta/featuretoggle/DefaultMultiSourceFeatureToggleService.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.befta.featuretoggle;
+
+import io.cucumber.java.Scenario;
+import uk.gov.hmcts.befta.exception.FeatureToggleCheckFailureException;
+import uk.gov.hmcts.befta.featuretoggle.launchdarkly.LaunchDarklyFeatureToggleService;
+
+public class DefaultMultiSourceFeatureToggleService implements FeatureToggleService<Scenario, ScenarioFeatureToggleInfo> {
+
+    public static final DefaultMultiSourceFeatureToggleService INSTANCE = new DefaultMultiSourceFeatureToggleService();
+    private static final String LAUNCH_DARKLY_FLAG = "FeatureToggle";
+
+    public static final String COLON = ":";
+    public static final String STRING_EQUALS = "=";
+
+    @Override
+    public ScenarioFeatureToggleInfo getToggleStatusFor(Scenario toggleable) throws FeatureToggleCheckFailureException {
+        ScenarioFeatureToggleInfo scenarioFeatureToggleInfo = new ScenarioFeatureToggleInfo();
+        // Examples : @FeatureToggle(LD:feature_id_1=on) @FeatureToggle(IAC:feature_id_2=off)
+        toggleable.getSourceTagNames().forEach(tag -> {
+            if (tag.contains(LAUNCH_DARKLY_FLAG)) {
+                String domain = null;
+                String id = null;
+                Boolean expectedStatus = null;
+                Boolean actualStatus;
+
+                domain = tag.contains(COLON) ? tag.substring(tag.indexOf("(") + 1, tag.indexOf(COLON)) : "LD";
+
+                FeatureToggleService service = getToggleService(domain);
+
+                if (!tag.contains(COLON) && !tag.contains(STRING_EQUALS)) {
+                    id = tag.substring(tag.indexOf("(") + 1, tag.indexOf(")"));
+                } else if (tag.contains(COLON) && !tag.contains(STRING_EQUALS)) {
+                    id = tag.substring(tag.indexOf(COLON) + 1, tag.indexOf(")"));
+                } else if (tag.contains(COLON) && tag.contains(STRING_EQUALS)) {
+                    id = tag.substring(tag.indexOf(COLON) + 1, tag.indexOf(STRING_EQUALS));
+                }
+
+                if (tag.contains(STRING_EQUALS)) {
+                    String expectedStatusString = tag.substring(tag.indexOf(STRING_EQUALS) + 1, tag.indexOf(")"));
+                    expectedStatus = expectedStatusString.equalsIgnoreCase("on");
+                    scenarioFeatureToggleInfo.addExpectedStatus(id, expectedStatus);
+                }
```

Comment:
This section could be nicer with regular expressions, but for now, it's good to go. We may refactor later if the tag syntax gets more complicated.

Reply:
We have sufficient coverage for the possible scenarios. However, a user still needs to be aware of the syntax of the annotations. We have integrated this code with AM repo, and it's working fine. 
If a user passes a junk, then any given framework would fail.